components/radio-button/README.md

```diff
@@ -156,4 +156,22 @@ const radioGroup = document.querySelector('gameface-radio-group');
 radioGroup.value = {{SOME_RADIO_BUTTON_VALUE}};
 ```
 
-If the value you've set doesn't match any of the available radio button values it will return a warning. 
\ No newline at end of file
+If the value you've set doesn't match any of the available radio button values it will return a warning. 
+
+### Checking Previous or Next Button
+
+You can select adjacent `<radio-button>` elements using the `<gameface-radio-group>` methods:
+`checkPrev()` and `checkNext()`.
+
+If there are one or more following elements with the `disabled` attribute, 
+the methods will check the next available one.

```

Comment:
```suggestion
If there are one or more following elements with the `disabled` attribute, the methods will check the next available one.
```

Reply:
Remember to run the transfer docs script to move any changes from the README to the docs